src/avram/tasks/db/verify_connection.cr

```diff
@@ -0,0 +1,20 @@
+class Db::VerifyConnection < LuckyCli::Task
+  summary "Verify connection to postgres"
+
+  def call
+    begin
+      DB.open(Avram::Migrator::Runner.database_url) do |db|
+      end
+    rescue PQ::ConnectionError | DB::ConnectionRefused
+      raise <<-ERROR
+      Unable to connect to postgres.
```

Comment:
I think it'd be good to also print which database it is and/or the url that was used so it is clear what is being used

Reply:
I have that on the line below it under the second help option. Does that work? If I print the whole URL, that may include the password, does that matter?

Reply:
I don't think so since script/setup should only be used in dev, but I suppose it is possible  that someone uses it in prod on accident. Maybe better to be safe than sorry. 

How about we just print the database it is using: https://github.com/luckyframework/avram/blob/3a6e5df4aac9239c71d733163f3cc0948433aeb8/src/avram.cr#L20 `Avram.settings.database_to_migrate`. And maybe we print out each section instead of a psql command, and we just filter out the password. So it'd look something like this:

```
Unable to connect to Postgres for database 'AppDatabase'

Connection info:

  * username: postgres
  * host: whatever
  * password: check config/database.cr

Try this...

  >  Check your database config in config/database.cr and make sure it is correct
  > Then run lucky db.verify_connection to make sure it can connect
```

Thoughts? I believe you can get connection info here https://github.com/luckyframework/avram/blob/3a6e5df4aac9239c71d733163f3cc0948433aeb8/src/avram/migrator/runner.cr#L15-L33

Reply:
And maybe include "make sure Postgres is running" 🤣